bio.html

```diff
@@ -0,0 +1,138 @@
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                    <span class="icon-bar"></span>
+                </button>
+                <a class="navbar-brand" href="#">PDG Poetry</a>
+            </div>
+            <!-- Collect the nav links, forms, and other content for toggling -->
+            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+                <ul class="nav navbar-nav">
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Services</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+            <!-- /.navbar-collapse -->
+        </div>
+        <!-- /.container -->
+    </nav>
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+      <div class="container">
+        <h1>Maya Angelou</h1>
+        <figure>
+          <img src="/media/maya-image.jpg" alt="" width="320" height="213">
+          <figcaption>Maya Angelou</figcaption>
+        </figure>
+        <br>
+        <p>Maya Angelou (born Marguerite Annie Johnson in April 4, 1928) was an American poet, memoirist, and civil rights activist. She published seven autobiographies, three books of essays, several books of poetry, and was credited with a list of plays, movies, and television shows spanning over 50 years. She received dozens of awards and more than 50 honorary degrees.[3] Angelou is best known for her series of seven autobiographies, which focus on her childhood and early adult experiences. The first, I Know Why the Caged Bird Sings(1969), tells of her life up to the age of 17 and brought her international recognition and acclaim.</p>
+
+<p>She became a poet and writer after a series of occupations as a young adult, including fry cook, sex worker, nightclub dancer and performer, cast member of the opera Porgy and Bess, coordinator for the Southern Christian Leadership Conference, and journalist in Egypt and Ghana during the decolonization of Africa. She was an actor, writer, director, and producer of plays, movies, and public television programs. In 1982, she earned the first lifetime Reynolds Professorship of American Studies at Wake Forest University in Winston-Salem, North Carolina. She was active in the Civil Rights movement and worked with Martin Luther King Jr. and Malcolm X. Beginning in the 1990s, she made around 80 appearances a year on the lecture circuit, something she continued into her eighties. In 1993, Angelou recited her poem "On the Pulse of Morning" (1993) at President Bill Clinton's inauguration, making her the first poet to make an inaugural recitation since Robert Frost at President John F. Kennedy's inauguration in 1961.</p>
+
+        <div class="work">
```

Comment:
My additions to [Issue #4](https://github.com/suwebdev/purple-haze/issues/4) start here: